src/System.Windows.Forms/tests/IntegrationTests/UIIntegrationTests/Infra/ControlTestBase.cs

```diff
@@ -252,6 +252,39 @@ protected async Task RunFormAsync<T>(Func<(Form dialog, T control)> createDialog
             await test.JoinAsync();
         }
 
+        protected async Task RunFormWithoutControlAsync(Func<PrintPreviewDialog> createForm, Func<PrintPreviewDialog, Task> testDriverAsync)
```

Comment:
No, no, no... This is a general purpose class and we mustn't hardcode any specific types here.
```suggestion
        protected async Task RunFormWithoutControlAsync<TForm>(Func<TForm> createForm, Func<TForm, Task> testDriverAsync) where TForm: Form
```